lib/cli/options.js

```diff
@@ -25,6 +25,25 @@ const EXIT_CODES = require('../shared/exit-codes');
 const packageJSON = require('../../package.json');
 
 /**
+ * @summary The required number of CLI arguments
+ * @constant
+ * @private
+ * @type {Number}
+ */
+const REQUIRED_NUMBER_OF_ARGUMENTS = 1;
+
+/**
+ * @summary The first index that represents an actual option argument
+ * @constant
+ * @private
+ * @type {Number}
+ *
+ * @description
+ * The first arguments are usually the program executable itself, etc.
```

Comment:
Not really. Consider that `argv` looks like this:

```sh
node bin/cli.js path/to/image.js <options>
```

The first two ones are not options.

Maybe the name is confusing? My intention is that `OPTIONS_INDEX_START` equals the first index of `argv` that is a option to the program.